README.md

```diff
@@ -101,51 +246,46 @@ iota.prepareTransfers(seed, transfers)
+We have some [basic contribution guidelines][contribution-guidelines] to keep our code base stable and consistent.
 
-Documentation of IOTA protocol and [`IRI`](https://github.com/iotaledger/iri) http API can be found on [docs.iota.works](https://docs.iota.works).
+### Cloning and bootstraping the repository on GitHub
 
-## Contributing
+1. Click the <kbd>Fork</kbd> button in the top-right corner
+2. Clone your fork and change directory into it
+3. Bootstrap your environment by doing the following:
 
-We thank everyone for their contributions. Here is quick guide to get started with iota.js monorepo:
+    ```bash
+    npm run init
+    ```
 
-### Clone and bootstrap
+This step will download all dependencies, build and link the packages together. iota.js uses [Lerna](https://lernajs.io/) to manage multiple packages. You can re-bootstrap your setup at any point with `lerna bootstrap` command.
 
-1. Fork the repo with <kbd>Fork</kbd> button at top right corner.
-2. Clone your fork locally and `cd` in it.
-3. Bootstrap your environment with:
-
-```
-npm run init
-```
+### Running tests
 
-This will install all dependencies, build and link the packages together. iota.js uses [Lerna](https://lernajs.io/) to manage multiple packages. You can re-bootstrap your setup at any point with `lerna bootstrap` command.
+Make your changes on a single package or across multiple packages and test the system by running the following from the root directory:
 
-### Run the tests
-
-Make your changes on a single or across multiple packages and test the system in integration. Run from the _root directory_:
-
-```
+```bash
 npm test
 ```
 
-To run tests of specific package just `cd` to the package directory and run `npm test` from there.
+To configure your editor to watch the tests running, add the `--watch` flag:
 
-You may also want to configure your editor to build the source uppon save and watch the tests running.
-Once building on save is setup, you can start watching tests with `npm test --watch` from each package directory.
+```bash
+npm test --watch`
+```
 
 ### Updating documentation
 
-Please update the documention when needed by editing [`JSDoc`](http://usejsdoc.org) annotations and running `npm run docs` from the _root directory_.
-
+Please update the documention when needed by editing [`JSDoc`](http://usejsdoc.org) annotations and running `npm run docs` from the root directory.
 
-## Reporting Issues
+## Joining the discussion
 
-Please report any problems you encouter during development by [opening an issue](https://github.com/iotaledger/iota.lib.js/issues/new).
+If you want to get involved in the community, need help with getting setup, have any issues related with the library or just want to discuss blockchain, distributed ledgers, and IoT with other people, feel free to join our [Discord][iota-discord].  
+You can also ask questions on our dedicated [IOTA Forum][iota-forum].
```

Comment:
let's just use [IOTA Discord](https://discord.gg/DTbJufa) and have discussion in one place?